subsys/mpsl/mpsl_init.c

```diff
@@ -159,9 +159,11 @@ static int mpsl_lib_init(const struct device *dev)
 	clock_cfg.skip_wait_lfclk_started =
 		IS_ENABLED(CONFIG_SYSTEM_CLOCK_NO_WAIT);
 
-#ifdef CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC
-	clock_cfg.rc_ctiv = MPSL_RECOMMENDED_RC_CTIV;
-	clock_cfg.rc_temp_ctiv = MPSL_RECOMMENDED_RC_TEMP_CTIV;
+#ifdef CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
```

Comment:
These values are not prefixed correctly. The code under ifdef won't compile.

Reply:
Thanks for pointing this out. It is now fixed